src/CoreBundle/private/modules/CMSModulePageTree/CMSModulePageTree.class.php

```diff
@@ -101,20 +104,18 @@ public function &Execute()
             $this->data['dataID'] = $this->global->GetUserData('id');
         }
 
-        if ($this->global->UserDataExists('rootID')) {
-            $nodeID = $this->global->GetUserData('rootID');
-            $this->data['rootID'] = $nodeID;
-            $this->iRootNode = $nodeID;
-            $this->GetRootNodeName($nodeID);
-            $this->LoadTreeState();
-            $this->aRestrictedNodes = $this->GetPortalNavigationStartNodes();
+        $nodeID = $inputFilterUtil->getFilteredGetInput('rootID', '99');
```

Comment:
99?

Reply:
Yes, this ID has been hard-coded for millennia. I moved the definition so that we at least don't need to save this ID in the new custom menu item for the navigation in DB.

Reply:
Still, wouldn't it be helpful to use a named constant?

Reply:
OK, added a constant and using it in several classes now.